inst/schema/scenarioResults.json

```diff
@@ -0,0 +1,51 @@
+{
+    "$schema": "http://json-schema.org/draft-04/schema#",
+    "type": "object",
+    "properties": {
+        "runId": {
+            "type": "string"
+        },
+        "costs": {
+            "type": "array",
+            "items": {
+                "$ref": "scenarioCost.json"
+            }
+        },
+        "capacities": {
+            "type": "array",
+            "items": {
+                "type": "object",
+                "properties": {
+                    "id": { "type": "string" },
+                    "value": { "type": "number" }
+                },
+                "additionalProperties": false,
+                "required": ["id", "value"]
+            }
+        },
+        "interventions": {
+            "type": "array",
+            "items": {
+                "type": "object",
+                "properties": {
+                    "id": { "type":  "string" },
+                    "start": { "type": "number" },
+                    "end": { "type": "number" }
+                },
+                "additionalProperties": false,
+                "required": [ "id", "start", "end" ]
+            }
+        },
```

Comment:
Same request as on the sample JSON above - could the response strategy and implementation level be added?